packages/terra-icon/scripts/src/generate-icon/renderJsx.js

```diff
@@ -19,25 +22,21 @@ const staticIcons = iconData
  * @param {object} icon Icon object that stores svg attributes, children, and name
  */
 const renderJsx = icon => new Promise((resolve, reject) => {
+  let templatePath;
   if (staticIcons.includes(icon.name)) {
-    fs.readFile(staticIconTemplatePath, 'utf-8', (error, text) => {
-      if (error) {
-        reject(error);
-      } else {
-        const compiled = template(text);
-        resolve(new ReactIcon(icon.name, compiled({ icon })));
-      }
-    });
+    templatePath = staticIconTemplatePath;
   } else {
-    fs.readFile(themeIconTemplatePath, 'utf-8', (error, text) => {
-      if (error) {
-        reject(error);
-      } else {
-        const compiled = template(text);
-        resolve(new ReactIcon(icon.name, compiled({ icon })));
-      }
-    });
+    templatePath = (iconsWithTooltip.includes(icon.name)) ? themeIconWithTooltipTemplatePath : themeIconTemplatePath;
```

Comment:
This logic is a bit confusing to me, does this mean that the static icons cannot/should not support tooltip?

Reply:
eIfu icon is themable icon and uses [theme template](https://github.com/cerner/terra-core/blob/dc294a9ec3f4f8825ecd45093a01510045db3e4a/packages/terra-icon/scripts/src/generate-icon/template.txt) so created tooltip template using theme template. If any requirement comes for static icons in future then we might need to create one more tooltip template for static Icons too.

Reply:
Let's rename `iconsWithTooltip` to `themeIconsWithTooltip` for clarity,  and then we can add logic for static icons if any requirement for static icons comes up in the future.

Reply:
renamed : 4fb01f556396ec875479704ccff8be8bf3dbea43